### ISSUE TITLE
Can't create Actions with latest update Mac Desktop v4.1.86 release

### ISSUE THREAD

**Issue by monocat** [FlutterFlow/flutterflow-issues#3749]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

As mentioned in title: Can't create Actions with latest update Mac Desktop v4.1.86 release. The behavior seems inconsistent across different widgets. For example, in one project, the Action functionality doesn't work on the iconButton widget, while in another project, it works fine with the same widget. Also, noticed the issue might be related when duplicating a widget and then attempting to apply an Action.

### Expected Behavior

Add Action item.

### Steps to Reproduce

1. Update Mac Desktop app to v4.1.86
2. Add a iconButton widget
3. Duplicate iconButton widget
4. Click to create Actions

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT48l/KB249OxLlE7pD+acFKmikvKUc7RJ0a0O0aeE08J5TUB+x/evz4UBJoTbyndghcI1qjsGUzw62Mm9z2UfEoZDqAc4hizKtIcjqWeE2lMpCIPKiwWHRuGshWN26w4ZjQqh57HPNpSiVm12b7BvC9fBHUNK7HClcnJ/ikUYfXrgfuQ0SLb3kNh1JWeC/v

### Visual documentation

![actions failure](https://github.com/user-attachments/assets/c0ef227b-a488-4fe8-8a13-290497bfe9e1)

### Environment

```markdown
- FlutterFlow version: 4.1.86
- Platform: Mac Desktop app
- Browser name and version:
- Operating system and version affected:
```

**Comment by htahta103** [comment]
I got the same issue when adding a new action for a widget with no action. 
Working correctly when add new actions on Web

**Comment by hidech** [comment]
Same for me. This is extremely important. Please add the ability to access previous versions of FF Mac in case this happens again.

**Comment by Emvipy** [comment]
same here

**Comment by rzambroni** [comment]
Hi everyone,

Thank you for bringing this issue to our attention, and I apologize for the inconvenience. I've confirmed the problem and have now forwarded it to the team for further investigation.

We’ll keep you updated on the progress.

**Comment by labraxoid** [comment]
I confirm the bug
and the saddest thing is that there is no way to roll back to the previous version of the application, now we have to sit and wait for the fix

**Comment by duuetdev** [comment]
Got the same issue, it still works on the website though!

**Comment by ryutosh** [comment]
Same here.
The work has been stopped and we are really in need of help.
For the past six months, I feel like FF have been downgrading..

**Comment by labraxoid** [comment]
> Got the same issue, it still works on the website though!

I confirm that it works on the web, but this [issue](https://github.com/FlutterFlow/flutterflow-issues/issues/3748) does not allow us to move forward, both bugs appeared after the last FF update

**Comment by yashcx** [comment]
We're encountering same issues. It seems that FlutterFlow is not recognizing any new updates to the builds, and no changes are being pushed from the desktop app (in my case, a Mac). We heavily rely on the Desktop app.

**Comment by rzambroni** [comment]
Hi everyone,

Just wanted to let you know that a fix is on the way. We’re sorry for the trouble and really appreciate your patience while we get this sorted.

**Comment by Wojtek-G** [comment]
Issue Fixed on my macOS with FF v4.1.87

**Comment by labraxoid** [comment]
I confirm the bug was fixed in version 4.1.87

**Comment by monocat** [comment]
Thank you @rzambroni Did a quick QA, issue is resolved on my end. Closing the ticket.
